### ISSUE TITLE
Screen won't turn off when Idle.

### ISSUE THREAD

**Issue by Leonature26** [valinet/ExplorerPatcher#3146]
After many days of troubleshooting all signs point to this application. And when I tried uninstalling this it worked, the screen would turn off after 1min. Installed it again and it no longer turns off. Also when uninstalled the cmd "power cfg /requests" shows none but when installed it says "...\explorer.exe". I'm on the latest version of everything and I think it's always been like this since I've installed explorerpatcher.

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.

**Comment by Leonature26** [comment]
So, I did a lot of digging on google and did a lot of fixes found across various forums and the thing that I think fixed it for me was detecting some virus. I used windows defender at first then it got stuck so I downloaded a free antivirus from google (landed on malwarebytes) then it detected a bunch of files and registry malware. Now powercfg /requests shows none. Also upon uninstalling explorerpatcher before all this the issue persisted sometimes so I guess this app had nothing to do with this issue.
